UploadApi/Controllers/DownloadController.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * DownloadController.cs
+ * API controller exhibiting file download.
+ *
+   Copyright 2018 Spectre Team
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Spectre.UploadApi.Services;
+
+namespace Spectre.UploadApi.Controllers
+{
+    [Route("api/[controller]")]
+    public class DownloadController : Controller
```

Comment:
In Spectre project we have classes extends ApiController https://github.com/spectre-team/spectre/tree/master/src/Spectre/Controllers , here Controller, maybe should we use one method?
https://stackoverflow.com/questions/9494966/difference-between-apicontroller-and-controller-in-asp-net-mvc

Reply:
This is in different technology - .NET Core. I suppose we will migrate old repo for Core too.